.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on: push
+
+jobs:
+  build_test_lint:
+    name: Node ${{ matrix.node_version }} and ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node_version: ['8.16', 10, 12] # Node 8 version fix due to https://github.com/actions/setup-node/issues/27
+        os: [ubuntu-latest, windows-latest, macOS-latest]
```

Comment:
I think ubuntu is fine or any of them, perhaps windows only, UI runs on browser anyway.

Reply:
Yes, UI will run on browser, but contributors can develop on several different systems. I think it's not bad idea to execute CI in the three specified.